components/centraldashboard/config/centraldashboard-config.yaml

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 data:
+  settings: |-
```

Comment:
Does this file live alongside the source code in order to allow the developers to easily apply it and play around with it?

I'm expecting that the manifests will also define such a ConfigMap object. We will also need to update the manifests with this change right?

Reply:
Yeap this is just an example so that we can give people a quick reference on how to configure centraldashboard

And yes, we will need to update the manifests with this change as well. I will make sure to include this when we refactor the manifests in Jan

Reply:
ACK, thanks for confirming!